app/api/healthcareAuthorities/getCursorApi.ts

```diff
@@ -11,10 +11,6 @@ export type Cursor = {
   version: string;
   name: string;
   publish_date_utc: number;
-  info_website_url: string;
-  api_endpoint_url: string; // Where cursor is hosted
-  privacy_policy_url: string;
-  reference_website_url: string;
   notification_threshold_percent: number;
   notification_threshold_count: number;
```

Comment:
same as before, see this [change in spec](https://pathcheck.atlassian.net/wiki/spaces/TEST/pages/81231882/Hashing+of+GPS+points+-+Design#Notification-of-sensitivity-controls)